tests/acceptance/stepDefinitions/notificationsContext.js

```diff
@@ -30,7 +30,7 @@ When('the user declines all shares displayed in the notifications on the webUI',
 
 Given('app {string} has been enabled', function (app) {
   const headers = httpHelper.createAuthHeader('admin')
-  return fetch(client.globals.backend_url + '/ocs/v2.php/cloud/apps/' + app + '?format=json', {
+  return fetch(join(client.globals.backend_url, '/ocs/v2.php/cloud/apps/', app) + '?format=json', {
```

Comment:
Better to move url into separate variable

Reply:
fixed